### PR TITLE
add Zaragoza bikes demo

### PR DESCRIPTION
**Pull request by ramiroaznar**

### CODE REVIEW

File: 02-builder-analysis/exercises/zgz-bikes.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+* Click on "Bike anchors" layer.
+* Go to **`ANALYSIS`** tab.
+* Click on **`ADD ANALYSIS`**.
+* Select `Create areas of influence` analysis and click on **`ADD ANALYSIS`**.
+* Set `RADIUS` to `250` meters and `BOUNDARIES` to `Dissolved`, and click on **`APPLY`**.
+
+![aoi](imgs/zgzbikes/02-aoi.png)
+
+* In order to recover our points layer, go to the main menu and drag the layer source node out.
+
```
Comment (csobier) [inline 02-builder-analysis/exercises/zgz-bikes.md]: Maybe break down this step a bit?

"...go back to the LAYERS pane. Click the original source of data {filename} from the _Bike anchors_ and drag it" above (or below..."

File: 02-builder-analysis/exercises/zgz-bikes.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+![aoi](imgs/zgzbikes/03-node.png)
+
+* Rename the other layer to "AOI" to distinguish the areas of influences and the original points.
+
+
+## Intersect areas and city blocks <a name="intersect"></a>
+
+* Click on "Blocks" layer.
+* Click it's **`ANALYSIS`** tab, and click on **`ADD ANALYSIS`**. Select the `Intersect second layer` analysis and click **`ADD ANALYSIS`**.
+* In the `INTERSECT LAYER` field, select the `AOI` node
```
Comment (csobier) [inline 02-builder-analysis/exercises/zgz-bikes.md]: Click the ANALYSIS tab, not "it's"

File: 02-builder-analysis/exercises/zgz-bikes.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+* Let the rest of parameters as default.
+* Click on **`APPLY`**.
+
+![intersect](imgs/zgzbikes/04-intersect.png)
+
+* To improve the visualization, we will remove the fill from the "AOIs" layer.
+* Go back to the main menu and select "AOIs" layer.
+* Go to the **`STYLE`** tab, click on the color `FILL` and set the opacity parameter (`A`) to `0`).
```
Comment (csobier) [inline 02-builder-analysis/exercises/zgz-bikes.md]: Go back to the LAYERS pane and ...

File: 02-builder-analysis/exercises/zgz-bikes.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+* You can set the `STROKE` color to red.
+
+![style](imgs/zgzbikes/05-style.png)
+
+## Assign total population to city blocks <a name="do"></a>
+
+* Go to the main menu and click on the "Blocks" layer.
+* Click it's **`ANALYSIS`** tab, and click on **`+`** to add a new analysis to the workflow chain. Select the `Enrich with Data Observatory` analysis and click **`ADD ANALYSIS`**.
```
Comment (csobier) [inline 02-builder-analysis/exercises/zgz-bikes.md]: LAYERS pane, and be mindful of "it's", which means "it is". Best to just say "the ANALYSIS tab"

File: 02-builder-analysis/exercises/zgz-bikes.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+* Consider adding the following widgets:
+
+  * formula widget for the sum of `population` from `Age and Gender` analysis node,
+  * formula widget for the sum of `anclajesdisponibles` from the original `Bikes anchors` layer node,
+  * formula widget for the sum of `bicisdisponibles` from the original `Bikes anchors` layer node,
+  * histogram widget for `population` from `Age and Gender` analysis node,
+  * category widget for `title` from the original `Bikes anchors` layer node.
+* Click on **`CONTINUE`**.
+* Go back to the widgets layout and select the `title` category widget. Set the `OPERATION` to `SUM(bicisdisponibles)` in order to order it based on the number of available bikes. You can add a suffix.
+
```
Comment (csobier) [inline 02-builder-analysis/exercises/zgz-bikes.md]: WIDGETS pane (Same as LAYERS)

**Comment by csobier** [comment]
Looks ok @ramiroaznar ! Just made a few, minor suggestions.

**Comment by ramiroaznar** [comment]
Thanks @csobier :trumpet:

**Pull request merged**